lib/src/Uncertainty/Algorithm/Analytical/openturns/Analytical.hxx

```diff
@@ -103,12 +110,12 @@ protected:
   friend class Factory<Analytical>;
 
 private:
-
   OptimizationAlgorithm nearestPointAlgorithm_;
   RandomVector event_;
   Point physicalStartingPoint_;
   AnalyticalResult result_;
-
+  Scalar limitStateTolerance_ = 0.0;
```

Comment:
This is odd: if the default value is defined in the ResourceMap (to 0.01), you don't need to define a different default here.